### ISSUE TITLE
Keep the original file for torrent seeding 

### ISSUE THREAD

**Issue by runa** [petrkotek/chromecastize#63]
Hi! I didn't knew that chromecastize existed so I did it myself and used it for years; but luckily I found your implementation and that's what I'm using nowadays :)

I find important to keep the original file *without changing its name* to keep torrent-seeding it. 

I was wondering if there's any configuration option for this or if you would be interested in this PR.

We could add a '-chromecast' suffix to the filename, before the extension.

Thanks for all your work :))

**Comment by pfrenssen** [comment]
It is not really in scope of this program to do filename management, there are other tools that are very suitable for this, like for example [mv](https://www.gnu.org/software/coreutils/manual/html_node/mv-invocation.html).

It is usually [not a good idea](https://danluu.com/cli-complexity/) to add options that duplicate other commands.
 
If you want the transcoded file to have a `-chromecast` suffix, why don't you just do:

```
$ cp myfile.mkv myfile-chromecast.mkv && chromecastize myfile-chromecast.mkv --delete-on-success
```

**Comment by runa** [comment]
Well, since I saw some [filename management](https://github.com/petrkotek/chromecastize/blob/master/chromecastize.sh#L117) I thought another option of "keep the original filename and create a new file" (exactly the oposite of adding a .bak suffix) would be acceptable.

The workaround you propose could work, but I wanted to exploit chromecastize's support for input directories (instead of video files).

I'll try working on this and create a PR for you to evaluate; if you want.

Thanks

**Comment by runa** [comment]
Please see it here https://github.com/petrkotek/chromecastize/pull/64

**Comment by pfrenssen** [comment]
Fixed in #64
